vendor/composer/autoload_commands_classmap.php

```diff
@@ -6,4 +6,96 @@
 $baseDir = dirname($vendorDir);
 
 return array(
+    'CLI_Command' => $vendorDir . '/wp-cli/wp-cli/php/commands/src/CLI_Command.php', 
```

Comment:
@CodeProKid looks like you need to run `composer install --no-dev` and commit again